README.md

```diff
@@ -1243,6 +1244,47 @@ Computed fields work like any other field, though with `computed: true` property
 
 You can see examples of computed fields in the [OAuth2](#oauth2) or [Session Auth](#session) example sections.
 
+### Nested & Children (Line Item) Fields
+
+When your action needs to accept an array of items, you can include an input field with the `children` attribute. The `children` attribute accepts a list of [fields](https://zapier.github.io/zapier-platform-schema/build/schema.html#fieldschema) that can be input for each item in this array.  
+
+```js
+const App = {
+  //...
+  operation: {
+    //...
+    inputFields: [
+      {
+      key: 'lineItems',
+      required: true,
+      children: [
+          {
+          key: 'lineItemId',
```

Comment:
minor, but it looks like these somehow got formatted weird in the output. Not a blocker, but if it's obvious how to make them look nice, it's worth taking a second to fix.